eng/pipelines/libraries/run-test-ready-to-run.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  archType: ''
+  buildConfig: ''
+  osGroup: ''
+  variables: []
+
+steps:
+  - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -ci
+            -subset libs.tests -test
```

Comment:
Are we intending to run tests locally instead of on Helix? If so, we should remove the `helixQueuesTemplate` option at https://github.com/dotnet/runtime/pull/91229/files#diff-b7cad421e96ec3229e0085c3a9f978858632fb29a034d369281db897678636f1R63.

If not, we should use the `libraries/helix.yml` template instead of the `-test` flag.

Also, any reason that this needs to be a separate invocation of the build script from the one in `build-for-test-ready-to-run.yml`?

Also, we have the `global-build-step.yml` template, which we could use instead of manually invoking the build script here. That template would handle the `-arch` and `-os` flags for us.

Reply:
Thanks for the observations Jeremy. You bring up some interesting points I had not considered that will certainly help improve this PR:

> Are we intending to run tests locally instead of on Helix?

We want them to run on Helix. So I'll check out `libraries/helix.yml` as you suggested.

> Also, any reason that this needs to be a separate invocation of the build script from the one in build-for-test-ready-to-run.yml?

This was because originally, we needed to have the `packs` subset built before attempting to run the tests, in order to use the fully self-contained Crossgen2. This was to get around that error where it tried to run the `crossgen2` executable directly and failed saying `You must install .NET to run this application`.

However, with the most recent changes to the Crossgen2 infrastructure, I just confirmed that this is no longer the case. We can now run say `--subset clr+libs+libs.tests` in the same call without issues. Will adjust it in CI as well.